README.md

```diff
@@ -20,83 +20,46 @@
 
 ### Neccessary Libraries
 
-    -Used in Datasource and Database
+    Libraries not pre-packaged with Python
+    that need installation.
+
+        -colorama
         -requests
-        -json
         -numpy
-        -sqlite3
-        -Pandas
-    -Used in Puzzle Class
-        -random
-    -Used in User Interface
-        -abc
-    -Used in Commands
-        -enum
-    -Used in Terminal Interface
-        -colorama
-        -math
-    -Used in State
-        -json
-        -os
-    -Used in Game Controller
-        -random
-        -string
-    -Used in Setup
-        -os
-        -json
-        -time
-    
+        -pandas
+        -pytest
 
-### Instructions for Building and Playing: Notes
+### Python Version
 
-    Note 1: The program has a few libraries that need installation for proper function.
-            These are colorama, requests, numpy, and pandas
-
-    Note 2: There are two sets of instructions below, one for those who want to use the 
-            setup python script and another for those who don't.
-
-    Note 3: if there is not a saveFiles folder present in the working directory, create one for
-            the game to store saves into (must be named explicitly "saveFiles")
+    Tested on Python 3.10.9 and Python 3.11.1
 
 
 ### Instructions for Building using Setup
 
-    1. Run setup via command line, in the form (python setup.py)
-
-    2. setup.py will prompt the user whether or not they want to 
-       automatically install all need libraries listed in Note 1 above.
-       If so, the commands for pip installing the libraries will be run.
-       Then, shortly after, the database will be setup to work with random
-       words.
-
-       Side note: If you do not have the pip installer, well, installed,
-       https://pip.pypa.io/en/stable/installation/, could help
+    1. Run setup via command line, in the form (python setup.py (Mac/Linux)) (py setup.py (Windows))
+
+    2. setup.py will create a viurutal environment, instatiate databases,
+       and intall any required non-base modules/packages into the environment
+
+    3. Entering Virutal Environment
+            Linux/Mac
+                -use the command line command (source spell/bin/activate)
+            Windows
+                -use the command line command (spell/Scripts/activate) (Must use powershell)
+                3.2.
+                    -when using windows, execution policy for scripts must be changed
+                     from default to allow scripts to run like the activate script for the
+                     virtual environment. This is done by using the command 
+                     (Set-ExecutionPolicy -ExecutionPolicy RemoteSigned)
```

Comment:
You may want to move this to point 1. as that's where the script is actually run?

Reply:
Changed 3.2 to 0 on the instructions for setup, as this should be done before step 1.